CSS/home.css

```diff
@@ -0,0 +1,83 @@
+/* Featured Products  */
+
+#featured-categories{
+    max-width: 100%;
+    margin: 0 4rem;
+}
+
+#featured-categories .categories-wrapper .category{
+    max-width: 15%;
+    height: auto;
+    padding: 2rem;
+    margin: 2rem;
+    cursor: pointer;
+    flex: 1;
+    flex-grow: 1;
+}
+
+#featured-categories .categories-wrapper .category img{
+    height: auto;
+    min-height: 7rem;
+    min-width: 7rem;
+    width: 100%;
+}
+
+/* Banner  */
+
+#home-banner{
+    max-width: 100%;
+    height: fit-content;
+    position: relative;
+    margin: 0 4rem;
+}
+
+#home-banner .banner-content{
+    position: absolute;
+    top: 50%;
+    right: 40%;
+}
+
+#home-banner img{
+    height: auto;
+    width: 100%;
+}
+
+#home-banner .banner-content .title{
+    color: var(--primary-dark-color);
+}
+
+#home-banner .banner-content .info{
+    color: var(--primary-dark-color);
+}
+
+/* New Arrivals  */
+
+#new-arrivals{
```

Comment:
it's not a good practice to  use ID selectors

Reply:
Ok